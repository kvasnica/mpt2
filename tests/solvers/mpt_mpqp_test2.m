function mpt_mpqp_test2

% tests that we properly detect infeasible problems:
% http://control.ee.ethz.ch/~mpt/hg/mpt?cs=42c743cf45db

fprintf('This problem should be infeasible from the begining:\n\n');
load mpqp_infeasible
[Pn, Fi, Gi] = mpt_mpqp(Matrices);
mbg_asserttrue(~isfulldim(Pn));
mbg_asserttrue(isempty(Fi));
mbg_asserttrue(isempty(Gi));
