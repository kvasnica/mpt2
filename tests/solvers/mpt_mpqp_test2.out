This problem should be infeasible from the begining:

mpt_mpqp:  Infeasible optimization problem from the begining